### DISCUSSION TITLE
New to SumatraPDF - but the text looks blurry compared to Acrobat Reader

### DISCUSSION THREAD

**Discussion by winghun** [sumatrapdfreader/sumatrapdf#4521] [Q&A] [answered]
Hi, 

I really like SumatraPDF, its simplicity is exactly what I'm looking for, but I have an issue. The displayed text seems blurry compared to Acrobat. 

[Example](https://github.com/user-attachments/assets/01a06a29-853a-4bdd-85e5-82ff6d4d99c3)

They're both at 125% zoom level. In Acrobat the text looks definitely much clearer though. It's the same in every PDF I tried.

Any idea how to fix this? Thanks

**Comment by kjk** [comment, marked as answer]
Rendering is handled by mupdf library and that's just how it is. There is no way to change it.

**Reply by winghun** [reply]
Ah that's unfortunate.
